src/pages/Work.tsx

```diff
@@ -16,10 +20,14 @@ import Signup from './signup';
 import Trips from './trips';
 import TripsReceipt from './trips/TripsReceipt';
 import AuthenticatedRoute from 'HOCs/AuthenticatedRoute';
+import '../styled/customStyles.scss';
 
-import Header from 'components/work/Header';
 
-import '../styled/customStyles.scss';
+const HostWithZendesk = (props: RouterProps) => (
+  <ZendeskWebWidgetWrapper>
+    <Host {...props} />
+  </ZendeskWebWidgetWrapper>
+);
```

Comment:
Hm, thinking it could get unwieldy to include this level of detail here (e.g. when other routes need wrappers). I like that `ZendeskWebWidgetWrapper` is not included in `Host.tsx`, that's a good separation of concerns; maybe we can introduce an intermediary component to handle this wrapping external to `Work.tsx`?

I'm picturing:

* /host/HostPage.tsx (currently Host.tsx in this PR)
* /host/Host.tsx (ZendeskWebWidgetWrapper around HostPage)
* /host/index.ts (just exports Host.tsx)

Reply:
+1 very clean architecture!